docs/content/docs/operators/feature/vectorassembler.md

```diff
@@ -156,12 +168,13 @@ input_data_table = t_env.from_data_stream(
             [DenseVectorTypeInfo(), Types.DOUBLE(), SparseVectorTypeInfo()])))
 
 # create a vector assembler object and initialize its parameters
-vector_assembler = VectorAssembler() \
-    .set_input_cols('vec', 'num', 'sparse_vec') \
-    .set_output_col('assembled_vec') \
+vector_assembler = VectorAssembler()\
+    .set_input_cols('vec', 'num', 'sparse_vec')\
+    .set_output_col('assembled_vec')\
+    .set_input_sizes(2, 1, 5)\
```

Comment:
It might be better to change `)\` to `) \`, keeping the same as python example file.